core/src/main/thrift-gen-java/org/apache/accumulo/core/manager/thrift/FateOperation.java

```diff
@@ -34,6 +34,7 @@
   TABLE_OFFLINE(5),
   TABLE_MERGE(6),
   TABLE_DELETE_RANGE(7),
+  @Deprecated
```

Comment:
This is thrift generated file and will be clobbered.

Reply:
Yep, I missed that. Thanks.

Reply:
Fixed in 96bd003